mongomock/helpers.py

```diff
@@ -1,4 +1,4 @@
-from collections import OrderedDict
+from collections import OrderedDict, abc
```

Comment:
Unfortunately we're still compatible with python2, so that won't fly. Either wait for pymongo to drop support for python2 or add a guard so that code doesn't fail in python2.

Reply:
Thank you for the review. Any suggestions or an example code on how to add a guard?

Reply:
I have implemented a fix. I hope this time this will fly.